### PR TITLE
Feat/html structure

### PR DESCRIPTION
**Pull request by borislavkb**
- Add basic html structure and classes

- Add style-sheet and links to css

- Add JS functionality

- App is functional and branch is ready to be merged

### CODE REVIEW

File: bmi_index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const weightInput = document.querySelector("#weight");
+
+const bmiMessage = document.querySelector(".result");
+
+// const button = document.querySelector("button");
+
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
```
Comment (mpagels) [inline bmi_index.js]: Commented code ;-) You don't want that in your production branch ;-)

File: bmi_index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  bmiMessage.textContent = "Your BMI is:" + bmi;
+  if (+bmi < 18.5 || bmi > 30) {
+    bmiMessage.style.color = "red";
+  } else if (18.5 < +bmi <= 25) {
+    bmiMessage.style.color = "green";
+  } else if (25 < +bmi < 30) {
+    bmiMessage.style.color = "orange";
+  }
+  formElement.reset();
+  console.log("clicked");
```
Comment (mpagels) [inline bmi_index.js]: Here is a lonely console.log ;-)

File: bmi_index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
+
+formElement.addEventListener("submit", (event) => {
+  const bmi = (weightInput.value / heightInput.value ** 2) * 10000;
+  event.preventDefault();
+  bmiMessage.textContent = "Your BMI is:" + bmi;
+  if (+bmi < 18.5 || bmi > 30) {
```
Comment (mpagels) [inline bmi_index.js:13]: Why is there a `+` before the bmi?
But really clever to check here for  both conditions. I like it very much.

File: bmi_index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+// const button = document.querySelector("button");
+
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
+
+formElement.addEventListener("submit", (event) => {
+  const bmi = (weightInput.value / heightInput.value ** 2) * 10000;
```
Comment (mpagels) [inline bmi_index.js:10]: maybe there is a method to not get a very long bmi number?
For example right now I'm getting `27.75748721694668` as an result. `27.76` would be way better ;-)

File: index.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+body,
+h1 {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (mpagels) [inline index.css:7]: You're giving the `body` and the `h1` the same styling? Is this intended?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BMI Calculator</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header></header>
```
Comment (mpagels) [inline index.html:11]: Maybe this would be a better place for the `h1` element?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+      <section class="bmi-container">
+        <div class="bmi-box">
+          <h1 class="heading">BMI Calculator</h1>
+        </div>
+        <div class="bmi-form-div">
+          <form class="bmi-form" action="">
+            <label for="Height">Your height:</label>
+            <input type="text" id="height" name="185cm" />
+            <label for="Weight">Your weight:</label>
+            <input type="text" id="weight" name="85kg" />
```
Comment (mpagels) [inline index.html:22]: You're missing some opportunities here:

First the `for` attribute in the `label` element helps for `focusing` the corresponding `input`. Therefor the `input` needs the `id` and the `name` as same as the `for` value.
Then, when you click on the Label with the mouse or touch it, the input is focused.

Read more about here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

Second, you could provide a `placeholder` so the user knows what unit he or she has to input.

**Review by mpagels — CHANGES_REQUESTED** [review]
Very good Boris. I really like what I see here. I found some tiny issues. Pls review.

File: bmi_index.js (hunk 1 above)
Comment (borislavkb) [reply bmi_index.js]: Removed it! Thx !

File: bmi_index.js (hunk 4 above)
Comment (borislavkb) [reply bmi_index.js:10]: You're right! :) There is a method => bmi.toFixed(2); 
Implemented it

File: bmi_index.js (hunk 3 above)
Comment (borislavkb) [reply bmi_index.js:13]: wanted to be sure that bmi would end up a number. Removed it 👍

File: bmi_index.js (hunk 2 above)
Comment (borislavkb) [reply bmi_index.js]: A forgotten item. Removed it as well. Thx !

File: index.css (hunk 5 above)
Comment (borislavkb) [reply index.css:7]: I guess I was in a hurry of some sort. Separated those 🥇

File: index.html (hunk 6 above)
Comment (borislavkb) [reply index.html:11]: Agree. Moved the ```h1``` to header.

File: index.html (hunk 7 above)
Comment (borislavkb) [reply index.html:22]: ids and names on ```input```corrected. Added ```placeholder```on the missing fields.

**Review by borislavkb — COMMENTED** [review]
Removed comments from code. 
Added rounding to BMI number. 
Added missing attributed to ```input``` fields. 
Minor styling changes.

**Pull request merged**